atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepMetrics.java

```diff
@@ -15,38 +15,39 @@
  */
 package com.palantir.atlasdb.sweep;
 
+import java.util.Map;
+
+import com.codahale.metrics.Histogram;
 import com.codahale.metrics.Meter;
 import com.codahale.metrics.MetricRegistry;
+import com.google.common.collect.ImmutableMap;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
 import com.palantir.atlasdb.util.MetricsManager;
+import com.palantir.tritium.metrics.registry.MetricName;
 
 @SuppressWarnings("checkstyle:FinalClass")
 public class SweepMetrics {
```

Comment:
This class is doing things - counting continual sweep metrics and end-of-table sweep metrics.  Consider making 2 classes to capture these different requirements?

Reply:
In fact - you might even get away with a single class of which there are 2 instances - one for continual metrics and one for end-of-table.  You'd have to avoid metric name collisions and possibly some other problems but remove a lot of duplication.